modules/doc/content/application_usage/tracked_apps.md

```diff
@@ -1,44 +1,140 @@
-# Tracked Apps
+# Applications leveraging MOOSE
 
 MOOSE is designed for building custom applications, the following is a community driven list of applications.
 
 The purpose of these lists is two fold:
 
 1. Every MOOSE-based application is a library, thus it is natural to create new applications from existing code.
-1. The MOOSE developers strive to keep your applications up-to-date when major changes occur, this list indicates to the developers to consider your application.
+1. The MOOSE developers help keep many applications up to date. But we have to know your application exists to do so!
 
 ## Open-source Applications
 
-- [Pika](https://github.com/idaholab/pika) - Phase-field model for micro-structure evolution of ice
+### Multiphysics frameworks
+
+- [Cardinal](https://github.com/neams-th-coe/cardinal) - Integration of NekRS \& OpenMC with MOOSE for fusion and fission systems high-fidelity simulation
+- [SALAMANDER](https://github.com/idaholab/salamander) - Integration of Cardinal with TMAP8 for fusion blanket multiphysics simulations
+
+
+### Neutronics / Reactor / Particle Physics
+
+- [Aurora](https://aurora-multiphysics.github.io/aurora/) - Integration of OpenMC and MOOSE developed by the UKAEA
+- [Frog](https://github.com/idaholab/thermal_to_fusion_converter) - Integration of GEANT-4 for high-energy particle transport
+- [Moltres](https://github.com/arfc/moltres) - Molten salt reactor simulation
+- [Squirrel](https://github.com/philipJFpfahl/Squirrel) - Point kinetics with delayed neutron precursor advection in molten salt reactors
+
+
+### Thermal hydraulics and fluid dynamics
+
+- [OpenPronghorn](https://mooseframework.inl.gov/open_pronghorn/) - Coarse mesh computational fluid dynamics
+- [barnacle](https://github.com/idaholab/barnacle) - Shallow water equations in MOOSE
```

Comment:
I believe that @dschwen has a new shallow water equations implementation (coming)? The last commit to this repo was 6 years ago. I think it can be removed